src/org/isf/patient/test/Tests.java

```diff
@@ -0,0 +1,429 @@
+package org.isf.patient.test;
+
+
+import java.util.*;
+
+import org.isf.patient.model.Patient;
+import org.isf.patient.service.PatientIoOperations;
+import org.isf.utils.db.DbJpaUtil;
+import org.isf.utils.exception.OHException;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(locations = { "classpath:applicationContext.xml" })
+public class Tests 
+{	
+	private static DbJpaUtil jpa;	
+	private static TestPatient testPatient;
+	private static TestPatientContext testPatientContext;
+
+    @Autowired
+    PatientIoOperations patientIoOperation;
+
+    @BeforeClass
+    public static void setUpClass()  
+    {
+    	jpa = new DbJpaUtil();
+    	testPatient = new TestPatient();
+    	testPatientContext = new TestPatientContext();
+
+        return;
+    }
```

Comment:
All this file is wrongly formatted, @ArnaudFonzam please use https://github.com/informatici/openhospital-core/tree/develop/.ide-settings

Reply:
ok, the whole file has been deleted